elasticsearch/manifests/es-data-stateful.yaml

```diff
@@ -64,38 +60,43 @@ spec:
                     values:
                     - data
               topologyKey: kubernetes.io/hostname
-      terminationGracePeriodSeconds: 60
       containers:
       - name: es-data
-        image: quay.io/pires/docker-elasticsearch-kubernetes:6.3.2
+        image: docker.elastic.co/elasticsearch/elasticsearch:6.5.4
         env:
+        - name: ES_JAVA_OPTS
+          value: -Xms1536m -Xmx1536m
         - name: NAMESPACE
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        - name: NODE_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
-        - name: CLUSTER_NAME
+        - name: cluster.name
           value: myesdb
-        - name: NODE_MASTER
+        - name: discovery.zen.ping.unicast.hosts
```

Comment:
Can we point to the elastic docs for all of these?

Reply:
Done!